BLSQ/iaso
# Polio-1393 remove link to budget procedure

Link was crashing the page, both in table and cards (mobile) layouts

Related JIRA tickets : POLIO-1393

## Self proofreading checklist


## Changes
- Deleted the component and removed it where used
- Fixed color and poistioning of override step button

## How to test

Go to budget page, open a budgetn the page should not crash. Repeat with mobile layout

## Print screen / video
![Screenshot 2024-01-08 at 14 29 25](https://github.com/BLSQ/iaso/assets/38907762/499a488a-5f3d-4188-82fd-9d7cad73c735)
